lib/uplinkc/uplink_definitions.h

```diff
@@ -31,6 +31,19 @@ typedef struct Object           { long _handle; } ObjectRef;
 typedef struct Downloader       { long _handle; } DownloaderRef;
 typedef struct Uploader         { long _handle; } UploaderRef;
 typedef struct EncryptionAccess { long _handle; } EncryptionAccessRef;
+typedef struct CaveatPaths      { long _handle; } CaveatPathsRef;
+typedef struct CaveatNonce      { long _handle; } CaveatNonceRef;
+
+typedef struct Caveat {
```

Comment:
Ideally the user should be able to manually specify the caveat without needing to cross into Go code.

Reply:
I'm not sure what you mean with this comment and how to solve it. May you help out on this?